ibm/mas_devops/roles/kubeturbo/README.md

```diff
@@ -0,0 +1,81 @@
+Kubeturbo
+============
+Installs a kubeturbo instance and sends data to a defined Turbonomic instance under the cluster name.
+
+Role Variables
+--------------
+
+### cluster_name
+Required. The cluster name is required to install Kubeturbo.
+
+- Environment Variable: `CLUSTER_NAME`
+- Default: None
+
+### turbonomic_username: 
+Required. The username is required to access the Turbonomics instance.
+
+- Environment Variable: `TURBONOMIC_USERNAME`
+- Default: None
+
+### turbonomic_password: 
+Required. The password is required to access the Turbonomics instance.
+
+- Environment Variable: `TURBONOMIC_PASSWORD`
+- Default: None
+
+### turbonomic_route
+Required. The route is required to access the Turbonomics instance.
```

Comment:
This is the route to the turbonomics server instance I assume. The docs need to say that to help the user decide what to enter.